application/src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -4229,11 +4231,10 @@ input PlanVisitViaLocationInput {
   """
   minimumWaitTime: Duration = "PT0S"
   """
-  A list of stop locations. A stop location can be a quay, a stop place, a multimodal
-  stop place or a group of stop places. It is enough to visit ONE of the locations
-  listed.
+  A list of stop locations. A stop location can be a stop or a station.
+  It is enough to visit ONE of the locations listed.
   """
-  stopLocationIds: [String!]
+  stopLocationIds: [String!]!
```

Comment:
I think making this required is a mistake. If we introduce coordinates in the future, it's possible to either have these defined, coordinates defined or both.